scripts/deploy-validator.sh

```diff
@@ -34,13 +34,21 @@ if ! docker_compose_plugin_installed; then
     exit 1
 fi
 
+HOST=$1
+
 # Check if the host is provided as an argument
-if [ -z "$1" ]; then
-  echo "Usage: $0 <host>"
+if [ -z "$HOST" ]; then
+  echo "Usage: $0 <host> [--remote-image]"
```

Comment:
nit: Perhaps `--with-remote-image` is slightly more idiomatic for a `bool` flag?